int64 torque_controller_switch
---
int64 torque_controller_status